### PR TITLE
Aspect descriptions

### PR DESCRIPTION
**Pull request by giteden**
## Proposed Changes

- add descriptions to aspects
- add logos to aspects
- update aspects overview page (import latest docs)

### CODE REVIEW

File: scopes/dependencies/dependency-resolver/dependency-resolver.docs.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-description: Simplifies the dependency management of components in a Bit workspace.
-labels: ['core aspect', 'dependencies']
+description: Simplifies the dependency management components in a Bit workspace.
```
Comment (ranm8) [inline scopes/dependencies/dependency-resolver/dependency-resolver.docs.mdx:2]: Scalable and simple dependency resolution for components.

File: scopes/envs/envs/envs.docs.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Manages Environments and Environment Services
+description: A composable engine for the standardization of component development environments.
```
Comment (ranm8) [inline scopes/envs/envs/envs.docs.mdx:2]: A composable engine for creating and maintaining development environments.

File: scopes/generator/generator/generator.docs.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Generator for generating component templates
+description: Generates components from templates, for more efficient development of new components that adhere to best-practices and organization standards.
```
Comment (ranm8) [inline scopes/generator/generator/generator.docs.mdx:2]: A simple interface for generating component templates.

File: scopes/harmony/application/application.docs.mdx [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
+description: Builds and deploys independent components to make them available in runtime.
```
Comment (ranm8) [inline scopes/harmony/application/application.docs.mdx:2]: Build and compose apps in components.

File: scopes/pipelines/builder/builder.docs.mdx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-description: build components on isolated directories
+description: A build task manager for independent components.
```
Comment (ranm8) [inline scopes/pipelines/builder/builder.docs.mdx:2]: Component build pipelines. Customize and standardize your component build process.

File: scopes/preview/preview/preview.docs.mdx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Handles the bundlings and rendering of component compositions and documentations
+description: Provides a visually rich Component preview, in an isolated browser-based environment.
```
Comment (ranm8) [inline scopes/preview/preview/preview.docs.mdx:2]: Previews components in an isolated, browser-based environment.

File: scopes/scope/scope/scope.docs.mdx [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
-description: A server for hosting components
-labels: ['core aspect']
+description: A collaboration server for independent components
```
Comment (ranm8) [inline scopes/scope/scope/scope.docs.mdx:2]: A collaboration server for components. - No need for independent here

File: scopes/workspace/variants/variants.docs.mdx [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
-description: teambit.workspace/variants enables cascading configurations by selection of component groupings or sets in the workspace and applying configurations to these cascading groups
+description: Enables the configuration of group of Components.
```
Comment (ranm8) [inline scopes/workspace/variants/variants.docs.mdx:2]: Configuration variants allow to configure a selected group of components.

File: scopes/workspace/workspace/workspace.docs.mdx [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
-description: A local workspace for component development and composition
-labels: ['core aspect']
+description: A component development workspace designed for the construction of distributed systems with a monolith-like dev experience.
```
Comment (ranm8) [inline scopes/workspace/workspace/workspace.docs.mdx:2]: A development workspace for distributed, component-driven software.

File: scopes/compositions/compositions/compositions.docs.mdx [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Renders component instances as part of different compositions.
+description: Visualizes components in their different variations, in isolated development sandbox
```
[the remark this answers is not in the document]
Comment (giteden) [reply scopes/compositions/compositions/compositions.docs.mdx:2]: done

File: scopes/defender/tester/tester.docs.mdx [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Provides a unified interface for testing components
+description: Provides a unified interface for testing Components of different types
```
[the remark this answers is not in the document]
Comment (giteden) [reply scopes/defender/tester/tester.docs.mdx:2]: done

File: scopes/dependencies/dependency-resolver/dependency-resolver.docs.mdx (hunk 1 above)
Comment (giteden) [reply scopes/dependencies/dependency-resolver/dependency-resolver.docs.mdx:2]: done

File: scopes/envs/envs/envs.docs.mdx (hunk 2 above)
Comment (giteden) [reply scopes/envs/envs/envs.docs.mdx:2]: done

File: scopes/generator/generator/generator.docs.mdx (hunk 3 above)
Comment (giteden) [reply scopes/generator/generator/generator.docs.mdx:2]: done

File: scopes/pipelines/builder/builder.docs.mdx (hunk 5 above)
Comment (giteden) [reply scopes/pipelines/builder/builder.docs.mdx:2]: done

File: scopes/preview/preview/preview.docs.mdx (hunk 6 above)
Comment (giteden) [reply scopes/preview/preview/preview.docs.mdx:2]: done

File: scopes/scope/scope/scope.docs.mdx (hunk 7 above)
Comment (giteden) [reply scopes/scope/scope/scope.docs.mdx:2]: done

File: scopes/workspace/variants/variants.docs.mdx (hunk 8 above)
Comment (giteden) [reply scopes/workspace/variants/variants.docs.mdx:2]: done

File: scopes/workspace/workspace/workspace.docs.mdx (hunk 9 above)
Comment (giteden) [reply scopes/workspace/workspace/workspace.docs.mdx:2]: done

File: scopes/component/dev-files/dev-files.docs.mdx [hunk 12]
```diff
@@ -0,0 +1,8 @@
+---
+description: Defines Component files of a certain pattern as development files
```
Comment (ranm8) [inline scopes/component/dev-files/dev-files.docs.mdx:2]: Classifies component files for development purposes

File: scopes/defender/linter/linter.docs.mdx [hunk 13]
```diff
@@ -1,6 +1,6 @@
 ---
-description: Provides a single interface for Component linting
-labels: ['core aspect', 'bit']
+description: Provides a unified interface for the linting of Components of different types
```
Comment (ranm8) [inline scopes/defender/linter/linter.docs.mdx:2]: A unified interface for component linting.

File: scopes/defender/formatter/formatter.docs.mdx [hunk 14]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (ranm8) [inline scopes/defender/formatter/formatter.docs.mdx:1]: A unified interface for formatting components

File: scopes/harmony/application/application.docs.mdx (hunk 4 above)
Comment (ranm8) [reply scopes/harmony/application/application.docs.mdx:2]: ?

File: scopes/compilation/compiler/compiler.docs.mdx [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Provides a single interface for Component compilation
+description: Provides a single interface for the compilation of Components of different types
```
Comment (ranm8) [inline scopes/compilation/compiler/compiler.docs.mdx:2]: A unified interface for component compilation.

File: scopes/harmony/application/application.docs.mdx (hunk 4 above)
Comment (giteden) [reply scopes/harmony/application/application.docs.mdx:2]: done

File: scopes/component/dev-files/dev-files.docs.mdx (hunk 12 above)
Comment (giteden) [reply scopes/component/dev-files/dev-files.docs.mdx:2]: done

File: scopes/defender/linter/linter.docs.mdx (hunk 13 above)
Comment (giteden) [reply scopes/defender/linter/linter.docs.mdx:2]: done

File: scopes/defender/formatter/formatter.docs.mdx (hunk 14 above)
Comment (giteden) [reply scopes/defender/formatter/formatter.docs.mdx:1]: done

File: scopes/compilation/compiler/compiler.docs.mdx (hunk 15 above)
Comment (giteden) [reply scopes/compilation/compiler/compiler.docs.mdx:2]: done

**Review by GiladShoham — APPROVED** [review]

**Pull request merged**